src/cluecode/finder_data.py

```diff
@@ -200,18 +217,24 @@ def set_from_text(text):
    .png
    .jpg
    .gif
+   .jpeg
```

Comment:
Can you elaborate why you made this addition and why this is related to spam Email detection?

Reply:
Since we're checking for junk suffixes, and we consider other image/gif formats to be candidates for junk suffixes, I assumed a .jpeg suffix would be considered here too, especially while classifying spam emails? Can revert it, if you disagree.